tests/test_kustomize_matrix.py

```
@@ -175,6 +184,136 @@ def test_unfold_expands_matrix_and_check_detects_stale_overlay(tmp_path):
+    assert rendered_component.count("# SPDX-License-Identifier") == 1
+    assert (
+        "# This comment is copied into the generated Component." in rendered_component
+    )
+    parsed_component = yaml.safe_load(rendered_component)
+    assert "setting: from-base" in parsed_component["patches"][0]["patch"]
+    assert 'replicas: "6"' in parsed_component["patches"][0]["patch"]
+    assert parsed_component["patches"][1]["path"] == "patch.yaml"
+    assert (component.parent / "patch.yaml").read_text(encoding="utf-8") == (
+        template / "patch.yaml"
+    ).read_text(encoding="utf-8")
+    build = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)